src/xrUICore/TrackBar/UITrackBar.cpp

```diff
@@ -290,6 +297,22 @@ void CUITrackBar::UpdatePos()
         pos.x = free_space - pos.x;
 
     m_pSlider->SetWndPos(pos);
+
+    if (m_static->IsEnabled())
+    {
+        string128 buff;      
+        if (m_b_is_float)
+        {
+            xr_sprintf(buff, (m_static_format == nullptr ? "%.1f" : m_static_format.c_str()), m_f_val);
+            m_static->TextItemControl()->SetTextST(buff);
+        }
+        else
+        {
+            string32 buff;
```

Comment:
The second variable named 'buff'??? And maybe use xr_string?

Reply:
Don't know what is better. Will fix buff.

Reply:
It's better to use simple type which `string32` is.

Reply:
std::string usually has small-string optimization technique (to avoid allocations if the length of the string is lesser than 16 bytes, for example). So, xr_string seems to be a good candidate here.